modules/common-fs2/src/main/scala/com/snowplowanalytics/snowplow/enrich/common/fs2/io/Source.scala

```diff
@@ -12,22 +12,32 @@
  */
 package com.snowplowanalytics.snowplow.enrich.common.fs2.io
```

Comment:
`io` is shadowing namespace of dependencies like `io.circe`. This confuses IDE when importing `io.*` in `test`.
I suggest changing it to `inout` or some other name that is not used as a top-level domain.